Description: Disable jnr-ffi native usage
 There is not a jnr-ffi native jar in Debian
Author: Miguel Landaeta <nomadium@debian.org>
Forwarded: no
Last-Update: 2017-02-01

--- jruby-9.1.6.0.orig/core/pom.xml
+++ jruby-9.1.6.0/core/pom.xml
@@ -638,6 +638,7 @@ DO NOT MODIFIY - GENERATED CODE
       <plugin>
         <artifactId>maven-dependency-plugin</artifactId>
         <executions>
+          <!--
           <execution>
             <id>unzip native</id>
             <phase>clean</phase>
@@ -659,6 +660,7 @@ DO NOT MODIFIY - GENERATED CODE
               </artifactItems>
             </configuration>
           </execution>
+          -->
         </executions>
       </plugin>
     </plugins>
@@ -1044,6 +1046,7 @@ DO NOT MODIFIY - GENERATED CODE
           <plugin>
             <artifactId>maven-dependency-plugin</artifactId>
             <executions>
+              <!--
               <execution>
                 <phase>process-classes</phase>
                 <goals>
@@ -1064,6 +1067,7 @@ DO NOT MODIFIY - GENERATED CODE
                   </artifactItems>
                 </configuration>
               </execution>
+              -->
             </executions>
           </plugin>
         </plugins>
